views/chainparameters.tmpl

```diff
@@ -29,70 +29,85 @@
-                            </tr>
-                            <tr>
-                                <td class="text-right pr-2 lh1rem nowrap p03rem0">RETARGET ADJUSTMENT FACTOR</td>
-                                <td class="lh1rem" >{{.RetargetAdjustmentFactor}}</td>
-                                <td class="lh1rem">RetargetAdjustmentFactor is the adjustment factor used to limit the minimum and maximum amount of adjustment that can occur between difficulty retargets.</td>
+                                <td class="lh1rem">Amount of time to generate each block.</td>
                             </tr>
                             <tr>
-                                <td class="text-right pr-2 lh1rem nowrap p03rem0">MAXIMUM BLOCK SIZES</td>
-                                <td class="lh1rem">{{.MaximumBlockSizes}} b</td>
-                                <td class="lh1rem">The maximum size a block can get to</td>
+                                <td class="text-right pr-2 lh1rem nowrap p03rem0">MAXIMUM BLOCK SIZE</td>
+                                <td class="lh1rem">{{.MaximumBlockSize}} b</td>
+                                <td class="lh1rem">Size a block can get to</td>
                             </tr>
                             <tr>
                                 <td class="text-right pr-2 lh1rem nowrap p03rem0">COINBASE MATURITY</td>
                                 <td class="lh1rem">{{.CoinbaseMaturity}} blocks</td>
-                                <td class="lh1rem">The number of blocks required before newly mined</td>
+                                <td class="lh1rem">Number of blocks required before newly mined</td>
                             </tr>
                             <tr>
-                                <td class="text-right pr-2 lh1rem nowrap p03rem0">SStxChange MATURITY</td>
+                                <td class="text-right pr-2 lh1rem nowrap p03rem0">TICKET PURCHASE CHANGE MATURITY</td>
                                 <td class="lh1rem">{{.SStxChangeMaturity}}</td>
                                 <td class="lh1rem">Maturity for spending SStx change outputs</td>
                             </tr>
+                            <tr>
+                                <td class="text-right pr-2 lh1rem nowrap p03rem0">TICKET MATURITY</td>
+                                <td class="lh1rem">{{.TicketMaturity}} blocks</td>
+                                <td class="lh1rem">Number of blocks for tickets to mature</td>
+                            </tr>
                             <tr>
                                 <td class="text-right pr-2 lh1rem nowrap p03rem0">TICKET POOL SIZE</td>
                                 <td class="lh1rem">{{.TicketPoolSize}}</td>
-                                <td class="lh1rem">This denotes the number of possible buckets/number of different ticket numbers</td>
+                                <td class="lh1rem">Number of possible number of different ticket numbers</td>
```

Comment:
As I mentioned before, we really want to show the value of TicketPoolSize x TicketsPerBlock.  No one will get the ticket numbers part.  With this being 40960, it can simply be described as "Target size of ticket pool"